cypress/integration/userCanPostNewRequest.feature.js

```diff
@@ -60,7 +60,7 @@ describe("User can post new reQuest, after navigating to creation form", () => {
       cy.get("#submit-btn").contains("Submit").click();
       cy.get("#title").then(($title) => {
         expect($title[0].validationMessage).to.eq(
-          "Please fill out this field."
+          "Please fill in this field."
```

Comment:
For some reason this was passing when someone pushed it, but it wasn't on my fresh development branch. I am very confused about this one.

Reply:
Is this dependent on which browser you are running?

Reply:
This should say 'fill out', because that's what Semaphore gets! So yes, it's some browser thing...